include/psa/crypto_values.h

```diff
@@ -1440,6 +1440,19 @@
  */
 #define PSA_KEY_LIFETIME_PERSISTENT             ((psa_key_lifetime_t)0x00000001)
 
+/** The minimum value for a key identifier chosen by the application.
+ */
+#define PSA_KEY_ID_USER_MIN                     ((psa_key_id_t)0x00000000)
```

Comment:
Needs a `u` suffix. Same for the following three constants

Reply:
Does this actually matter, given that the constants are being cast? If it does we need to change all the literals in the specification, not just these.

Reply:
Can be changed later (in a different PR), if deemed necessary.